src/main/java/org/candlepin/subscriptions/tally/filler/TallyReportDataPointAdapter.java

```diff
@@ -34,7 +34,8 @@ public boolean itemIsLarger(TallyReportDataPoint oldData, TallyReportDataPoint n
   @Override
   public TallyReportDataPoint createDefaultItem(
       OffsetDateTime itemDate, TallyReportDataPoint previous, boolean useRunningTotalFormat) {
-    var point = new TallyReportDataPoint().date(itemDate).hasData(false).value(0);
+    // Mark the datapoint as having data since this is a running total
+    var point = new TallyReportDataPoint().date(itemDate).hasData(true).value(0);
```

Comment:
We need `hasData=false` when `useRunningTotalFormat=false`:
```suggestion
    var point = new TallyReportDataPoint().date(itemDate).hasData(useRunningTotalFormat).value(0);
```

Reply:
I didn't see any other problem than ^
I tested for `use_running_totals_format==true` and it returns `has_data: true` and no future date
But when I tested `use_running_totals_format==false` and it returns `has_data: true` instead of `useRunningTotalFormat` with future dates